tests/testing/__init__.py

```diff
@@ -17,18 +17,15 @@ def requirements(reqs):
 
 
 def run(*cmd, **env):
+    from os import environ
+    new_env = environ.copy()
     if env:
-        from os import environ
-        tmp = env
-        env = environ.copy()
-        env.update(tmp)
-    else:
-        env = None
+        new_env.update(env)
 
```

Comment:
None should be equivalent to environ.copy()

Please try to revert, and comment if can't.